trieste/acquisition/optimizer.py

```diff
@@ -129,27 +140,22 @@ def generate_continuous_optimizer(
 
     We advise the user to either use the default `NUM_SAMPLES_MIN` for `num_initial_samples`, or
     `NUM_SAMPLES_DIM` times the dimensionality of the search space, whichever is smaller.
+    Similarly, for `num_optimization_runs`, we recommend using `NUM_RUNS_DIM` times the
+    dimensionality of the search space.
 
-    This optimizer supports Scipy's L-BFGS-B optimizer (used via GPflow's Scipy optimizer wrapper),
-    which optimizes directly within and up to the bounds of the search space.
+    This optimizer supports Scipy's L-BFGS-B optimizer. We run `num_optimization_runs` separate
```

Comment:
```suggestion
    This optimizer uses Scipy's L-BFGS-B optimizer. We run `num_optimization_runs` separate
```
??? It's not really optional, is it?